src/meshapi/util/uisp_import/sync_handlers.py

```diff
@@ -0,0 +1,301 @@
+                    f"Azimuth defaulted to {sector.azimuth} degrees. Device name did not indicate "
+                    f"a cardinal direction. Please provide a more accurate value if available"
+                )
+
+            notify_admins_of_changes(
+                sector,
+                [
+                    azimuth_message,
+                    (
+                        f"Guessed coverage width of {sector.width} degrees from device type. "
+                        f"Please provide a more accurate value if available"
+                    ),
+                    f"Set default radius of {sector.radius} km. Please correct if this is not accurate",
+                ],
+                created=True,
+            )
+        else:
+            device = Device(**device_fields)
+            device.save()
+
+
+def import_and_sync_uisp_links(uisp_links: List[UISPDataLink]) -> None:
+    uisp_session = get_uisp_session()
+    for uisp_link in uisp_links:
+        uisp_uuid = uisp_link["id"]
+        if not uisp_link["from"]["device"]:
+            logging.warning(
+                f"During UISP link import link with UISP ID {uisp_uuid} was skipped "
+                f"because the data in UISP does not indicate what device this link connects from"
+            )
+            continue
+
+        uisp_from_device_uuid = uisp_link["from"]["device"]["identification"]["id"]
+        try:
+            uisp_from_device = Device.objects.get(uisp_id=uisp_from_device_uuid)
+        except Device.DoesNotExist:
+            logging.warning(
+                f"During UISP link import link with UISP ID {uisp_uuid} was skipped "
+                f"because the data in UISP references a 'from' device (UISP ID {uisp_from_device_uuid}) "
+                f"which we do not have in our database (perhaps it was skipped at device import time?)"
+            )
+            continue
+
+        if not uisp_link["to"]["device"]:
+            logging.warning(
+                f"During UISP link import link with UISP ID {uisp_uuid} was skipped "
+                f"because the data in UISP does not indicate what device this link connects to"
+            )
+            continue
+
+        uisp_to_device_uuid = uisp_link["to"]["device"]["identification"]["id"]
+        try:
+            uisp_to_device = Device.objects.get(uisp_id=uisp_to_device_uuid)
+        except Device.DoesNotExist:
+            logging.warning(
+                f"During UISP link import link with UISP ID {uisp_uuid} was skipped "
+                f"because the data in UISP references a 'to' device (UISP ID {uisp_to_device_uuid}) "
+                f"which we do not have in our database (perhaps it was skipped at device import time?)"
+            )
+            continue
+
+        if uisp_link["state"] == "active":
+            uisp_status = Link.LinkStatus.ACTIVE
+        else:
+            uisp_status = Link.LinkStatus.INACTIVE
+
+        uisp_link_type = get_link_type(uisp_link)
```

Comment:
TODO: This type can be incorrect when links get added manually in the UISP web console. Notably this results in fiber and VPN links loading as "ethernet"

To make things worse, if this is a new link, we won't even tell the admins about it so they can fix it....